### DISCUSSION TITLE
storage/hwameistor/install/deploy-image/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1279] [Announcements]
# storage/hwameistor/install/deploy-image/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/install/deploy-image/

**Comment by FloatXD** [comment]
新版本镜像仓库地址切换
```
helm install hwameistor ./hwameistor \
    -n hwameistor --create-namespace \
    --set global.k8sImageRegistry=m.daocloud.io/registry.k8s.io \
    --set global.hwameistorImageRegistry=ghcr.m.daocloud.io
```

**Reply by windsonsea** [reply]
欢迎直接提 PR 哇 :)

**Comment by WANG0608GitHub** [comment]
有关hwameistor的安装，请参考以下两个页面：
- [通过 HwameiStor-Operator 安装](https://docs.daocloud.io/storage/hwameistor/install/deploy-helm/)
- [通过界面安装](https://docs.daocloud.io/storage/hwameistor/install/deploy-ui/)
